include/net/tftp.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file tftp.h
+ *
+ */
+
+#ifndef ZEPHYR_INCLUDE_NET_TFTP_H_
+#define ZEPHYR_INCLUDE_NET_TFTP_H_
+
+#include <zephyr.h>
+#include <net/socket.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct tftpc {
+	u8_t   *user_buf;
+	u32_t  user_buf_size;
+};
+
+/* Name: tftp_get
+ * Description: This function gets "file" from the remote server.
+ */
+int tftp_get(struct sockaddr_in *server, struct tftpc *client,
```

Comment:
Commenting this although you closed it. I would prefer that we have the IP address in the API in protocol agnostic way, so `struct sockaddr *server` here. Then it is easier to add IPv6 support too.

Reply:
Thanks.. I'm just going to add IPv6 support right now instead of leaving it as a TODO for future. Shouldn't be a big deal as we're at the application level.

Reply:
Hi, Is there any update on IPV6?

Reply:
Yeah, I haven't worked on this since it first went in. Feel free to add / improve support wherever possible.